source/models/__mocks__/mock.ts

```diff
@@ -0,0 +1,15 @@
+import { jest } from '@jest/globals';
+
+export const mock = {
+    request: () => ({
```

Comment:
I think you do not need this. All places where you use it you can replace with {} as Request

Reply:
removed